charts/vmclarity/templates/apiserver/_helpers.tpl

```diff
@@ -0,0 +1,22 @@
+{{/*
+Base name for the apiserver components
+*/}}
+{{- define "vmclarity.apiserver.name" -}}
+{{ include  "vmclarity.names.fullname" . }}-apiserver
```

Comment:
maybe we need to truncate the full name to `63-len("-apiserver")`
Since we are doing
```
{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
```
but then we are adding more suffixes